npm/cypress-schematic/src/schematics/ng-generate/cypress-ct-tests/index.spec.ts

```diff
@@ -27,11 +24,13 @@ describe('ng-generate @cypress/schematic:specs-ct', () => {
   }
 
   beforeEach(async () => {
-    appTree = await schematicRunner.runExternalSchematicAsync('@schematics/angular', 'workspace', workspaceOptions).toPromise()
-    appTree = await schematicRunner.runExternalSchematicAsync('@schematics/angular', 'application', appOptions, appTree).toPromise()
+    appTree = await schematicRunner.runExternalSchematic('@schematics/angular', 'workspace', workspaceOptions)
+    appTree = await schematicRunner.runExternalSchematic('@schematics/angular', 'application', appOptions, appTree)
   })
 
   it('should create cypress component tests alongside components', async () => {
-    return schematicRunner.runSchematicAsync('specs-ct', { project: 'sandbox' }, appTree).pipe(take(1)).subscribe((tree: UnitTestTree) => expect(tree.files).to.contain('/projects/sandbox/app/src/app.component.cy.ts'))
+    const tree = await schematicRunner.runSchematic('specs-ct', { project: 'sandbox' }, appTree)
```

Comment:
this test was previously passing when it shouldn't have. I cannot mock the file system easily, so I added a `projects/sandbox/src` with a `.gitkeep` to make this test mock generate the cypress test correctly.